content/docs/command-reference/commit.md

```diff
@@ -42,10 +42,10 @@ scenarios are further detailed below.
 
 - In cases where we have previously executed a stage (either by writing
   `dvc.yaml` manually and using `dvc repro`, or with `dvc run`), but later
-  notice that some of the output files or directories it creates, which are
-  already in the <abbr>workspace</abbr>, are missing from `dvc.yaml` (`outs`
-  field). We can
-  [add missing outputs to an existing stage](/docs/user-guide/how-to/add-output-to-stage)
+  notice that some of the existing dependencies, or output files/directories it
+  creates, which are already in the <abbr>workspace</abbr>, are missing from
```

Comment:
"dependencies it creates" ? Again, let's think the changes through please.
Also, no comma was needed here.

Reply:
Pushed new update to clarify this.

Reply:
Also should title of the doc be same as label? https://github.com/iterative/dvc.org/pull/1914#discussion_r521781518

Reply:
What doc? What label? Sorry, didn't get that Q.

Reply:
I meant that title of how-to doc should be same as label "Add Tracked Data to a Stage"? Currently the title is "Add Dependencies or Outputs to a Stage".

Reply:
I think we discussed that before. The label is shorter so it fits in the nav. Please always check the docs you're writing on the dev environment, BTW. You will notice important details.

If you have a better suggestion for title or label, please open a separate comment on the corresponding lines of code. Resolcing this. Thanks